source/fe/fe_dgq.cc

```diff
@@ -656,10 +657,17 @@ FE_DGQ<dim, spacedim>::memory_consumption () const
 
 
 
+// ------------------------------ FE_DGQArbitraryNodes -----------------------
+
 template <int dim, int spacedim>
 FE_DGQArbitraryNodes<dim,spacedim>::FE_DGQArbitraryNodes (const Quadrature<1> &points)
-  : FE_DGQ<dim,spacedim>(points)
-{}
+  : FE_DGQ<dim,spacedim>(Polynomials::generate_complete_Lagrange_basis(points.get_points()))
```

Comment:
Ah, I see that the functionality has moved here. I think that ought to be good enough -- the original functionality has not disappeared.